src/animations/mviz/main.c

```diff
@@ -68,7 +68,7 @@ void ckb_init(ckb_runctx* context){
 
 void ckb_parameter(ckb_runctx* context, const char* name, const char* value){
     CKB_PARSE_AGRADIENT("color", &animcolor){}
-	CKB_PARSE_BOOL("power", &power);
+	CKB_PARSE_BOOL("power", &power){};
```

Comment:
What's up with the formatting here?

Reply:
Not really sure, it looks in line when cloned.

Reply:
L70 is indented with spaces, L71 with tabs. Depending on your editor, it will show a different indentation than GitHub (which uses 8 spaces for tabs).

I can give you a hint for how to let this show in vim, if you want. 😏

Reply:
You are right, sorry I did the quick patch at work in Nano on a machine I wasn't familiar with. I'll fix this now.

Reply:
It's been a while, fleischie. Glad you're still hanging around.